infra/cifuzz/cifuzz.py

```diff
@@ -0,0 +1,167 @@
+    if not os.path.exists(git_workspace):
+      os.mkdir(git_workspace)
+    out_dir = os.path.join(os.environ['GITHUB_WORKSPACE'], 'out')
+    if not os.path.exists(out_dir):
+      os.mkdir(out_dir)
+  else:
+    return Status.ERROR.value
+
+  # Change to oss-fuzz main directory so helper.py runs correctly.
+  if os.getcwd() != helper.OSSFUZZ_DIR:
+    os.chdir(helper.OSSFUZZ_DIR)
+
+  if args.command == 'build_fuzzers':
+    return build_fuzzers(args, git_workspace, out_dir).value
+  if args.command == 'run_fuzzers':
+    return run_fuzzers(args, out_dir).value
+  return Status.SUCCESS.value
+
+
+def build_fuzzers(args, git_workspace, out_dir):
+  """Builds all of the fuzzers for a specific OSS-Fuzz project.
+
+  Args:
+    args: List of args passed in to cifuzz.py build_fuzzers parser.
+    git_workspace: The location in the shared volume to store git repos.
+    out_dir: The location in the shared volume to store output artifacts.
+
+  Returns:
+    A Status enum representing the state of the build.
+  """
+  # TODO: Modify build_specified_commit function to return src dir.
+  src = '/src'
+
+  inferred_url, oss_fuzz_repo_name = build_specified_commit.detect_main_repo(
+      args.project_name, repo_name=args.github_repo_name)
+  if not inferred_url or not oss_fuzz_repo_name:
+    print('Error: Repo URL or name could not be determined.', file=sys.stderr)
+
+  # Checkout projects repo in the shared volume.
+  build_repo_manager = repo_manager.RepoManager(inferred_url,
+                                                git_workspace,
+                                                repo_name=oss_fuzz_repo_name)
+  try:
+    build_repo_manager.checkout_commit(args.commit_sha)
+  except repo_manager.RepoManagerError:
+    print('Error: Specified commit does not exist.', file=sys.stderr)
+    return Status.ERROR
+
+  command = [
+      '--cap-add', 'SYS_PTRACE', '--volumes-from',
+      utils.get_container(), '-e', 'FUZZING_ENGINE=libfuzzer', '-e',
+      'SANITIZER=address', '-e', 'ARCHITECTURE=x86_64',
+      'gcr.io/oss-fuzz/%s' % args.project_name, '/bin/bash', '-c',
+      'rm -rf /src/yara/* && cp -r {0} {1} && compile && cp -r {2} {3}'.format(
+          os.path.join(git_workspace, '.'), src, '/out/.', out_dir)
+  ]
+  if helper.docker_run(command):
+    print('Error: Building fuzzers failed.', file=sys.stderr)
+    return Status.ERROR
+  return Status.SUCCESS
+
+
+def run_fuzzers(args, out_dir):
+  """Runs a all fuzzers for a specific OSS-Fuzz project.
+
+  Args:
+    args: List of args passed in to cifuzz.py run_fuzzers parser.
+    out_dir: The location in the shared volume to store output artifacts.
+
+  Returns:
+    A Status enum representing the state of the run.
+  """
+  fuzzer_paths = utils.get_fuzz_targets(out_dir)
+  if not fuzzer_paths:
+    print('Error: No fuzzers were found in out directory.', file=sys.stderr)
+    return Status.ERROR
+
+  fuzzer_timeout = int(int(args.fuzz_time) / len(fuzzer_paths))
```

Comment:
rather than the outermost int here, use // for the division.

Reply:
Done.